### PR TITLE
Mock Swiping Controller is created

### PR DESCRIPTION
**Pull request by Riceqrisp**
Controller is checking token and returning 200 or 403

### CODE REVIEW

File: GiTinder/Controllers/SwipesController.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                responseBody = new OKResponseBody("ok", "success");
+                return StatusCode(200, responseBody);
+                if(direction == "right")
+                {
+                   //save swipe with both user ids as a match.
+                }
+                else
+                {
+                    //give show next user
+                }
```
Comment (adamgyulavari) [inline GiTinder/Controllers/SwipesController.cs]: @Riceqrisp this if-else is not needed

File: GiTinder/Controllers/UsersController.cs [hunk 2]
```diff
@@ -32,14 +32,19 @@ public async Task<GeneralApiResponseBody> Login([FromBody] LoginRequestBody logi
                 Response.StatusCode = 400;
                 responseBody =
                     String.IsNullOrEmpty(username) ?
+<<<<<<< HEAD
+                    new ErrorResponseBody("username is missing!") :
+=======
                     new ErrorResponseBody("username is missing!") : 
+>>>>>>> 78139706e305e548578182a508e42d27638a2e25
```
Comment (adamgyulavari) [inline GiTinder/Controllers/UsersController.cs]: @Riceqrisp conflict still here

File: GiTinder/Views/Shared/_ValidationScriptsPartial.cshtml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+            asp-fallback-test="window.jQuery && window.jQuery.validator"
+            crossorigin="anonymous"
+            integrity="sha384-rZfj/ogBloos6wzLGpPkkOr/gpkBNLZ6b6yLy4o+ok+t/SAKlL5mvXLr0OXNi1Hp">
+    </script>
+    <script src="https://ajax.aspnetcdn.com/ajax/jquery.validation.unobtrusive/3.2.9/jquery.validate.unobtrusive.min.js"
+            asp-fallback-src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.min.js"
+            asp-fallback-test="window.jQuery && window.jQuery.validator && window.jQuery.validator.unobtrusive"
+            crossorigin="anonymous"
+            integrity="sha384-ifv0TYDWxBHzvAk2Z0n8R434FL1Rlv/Av18DXE43N/1rvHyOG4izKst0f2iSLdds">
+    </script>
```
Comment (adamgyulavari) [inline GiTinder/Views/Shared/_ValidationScriptsPartial.cshtml]: @Riceqrisp what's this?

File: GiTinder/Views/Swipes/Create.cshtml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+﻿@model GiTinder.Models.Swipe
+
+@{
+    ViewData["Title"] = "Create";
+}
+
+<h2>Create</h2>
+
```
Comment (adamgyulavari) [inline GiTinder/Views/Swipes/Create.cshtml]: @Riceqrisp and this?

File: GiTinder/Views/Swipes/Delete.cshtml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        <dd>
+            @Html.DisplayFor(model => model.Timestamp)
+        </dd>
+    </dl>
+    
+    <form asp-action="Delete">
+        <input type="hidden" asp-for="SwipingUserId" />
+        <input type="hidden" asp-for="SwipedUserId" />
+        <input type="submit" value="Delete" class="btn btn-default" /> |
+        <a asp-action="Index">Back to List</a>
```
Comment (adamgyulavari) [inline GiTinder/Views/Swipes/Delete.cshtml]: @Riceqrisp and these all? :D

File: GiTinder/appsettings.json [hunk 6]
```diff
@@ -5,7 +5,11 @@
...
-  "ConnectionStrings": {
-    "GiTinderContextMSSqlDb": "Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=novelgitinder;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
-  }
+    "ConnectionStrings": {
+        "GiTinderContextMSSqlDb": "Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=novelgitinder;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
+    }
 }
+//Local:
+    //Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=novelgitinder;Integrated Security=True;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False
+//Remote:
```
Comment (adamgyulavari) [inline GiTinder/appsettings.json]: @Riceqrisp we shouldnt put the remote version of this string anywhere near github

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: GiTinder/Views/Swipes/Delete.cshtml (hunk 5 above)
Comment (Riceqrisp) [reply GiTinder/Views/Swipes/Delete.cshtml]: No idea where it comes from :(

File: GiTinder/Models/ResponseBody.cs [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+using System.Threading.Tasks;
+
+namespace GiTinder.Models
+{
+    public class ResponseBody
+    {
+        [JsonProperty("status", Order =-2)]
+        public string Status { get; set; }
+    }
+}
```
Comment (adamgyulavari) [inline GiTinder/Models/ResponseBody.cs]: @Riceqrisp do you use this anywhere?

File: GiTinder/Models/Responses/OKResponseBody.cs [hunk 8]
```diff
@@ -16,5 +16,9 @@ public OKResponseBody(string message)
             Status = "ok";
             Message = message;
         }
+        public OKResponseBody()
+        {
+
+        }
```
Comment (adamgyulavari) [inline GiTinder/Models/Responses/OKResponseBody.cs]: @Riceqrisp do you use this anywhere?

File: GiTinder/Services/ProfileService.cs [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+{
+    public class ProfileService
+    {
+        private GiTinderContext _context;
+
+        public ProfileService(GiTinderContext context)
+        {
+            _context = context;
+        }
+        public List<User> ReturnNProfilesRandomly(string username, int numberOfProfiles)
```
Comment (adamgyulavari) [inline GiTinder/Services/ProfileService.cs]: @Riceqrisp let's call this `GetProfilesForUser`

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: GiTinder/Models/ResponseBody.cs (hunk 7 above)
Comment (Riceqrisp) [reply GiTinder/Models/ResponseBody.cs]: hmm... :D. Deleted.

File: GiTinder/Models/Responses/OKResponseBody.cs (hunk 8 above)
Comment (Riceqrisp) [reply GiTinder/Models/Responses/OKResponseBody.cs]: Not anymore.

File: GiTinder/Services/ProfileService.cs (hunk 9 above)
Comment (Riceqrisp) [reply GiTinder/Services/ProfileService.cs]: Changed

File: GiTinder/Services/GetProfilesForUser.cs [hunk 10]
```diff
@@ -9,11 +9,11 @@
 
 namespace GiTinder.Services
 {
-    public class ProfileService
+    public class GetProfilesForUser
```
Comment (adamgyulavari) [inline GiTinder/Services/GetProfilesForUser.cs]: @Riceqrisp the class can remain ProfileService

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: GiTinder/Services/GetProfilesForUser.cs (hunk 10 above)
Comment (Riceqrisp) [reply GiTinder/Services/GetProfilesForUser.cs]: Omg my bad. Corrected.

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**